spec/knockoff_spec.rb

```diff
@@ -127,5 +127,31 @@ class ActiveRecord::Relation
         end
       end
     end
+
+    context 'passwords' do
+      it 'should allow simple passwords in URI' do
+        ENV['KNOCKOFF_REPLICA1'] = "mysql2://user:just_regular_password@localhost/tmp/test_replica_db"
```

Comment:
@erkie as-is, these tests will have a global change and leave `ENV['KNOCKOFF_REPLICA1']` to this new value.

can you ensure that it gets set back to the original value after each test? see above for an example you can use.

Reply:
@sgringwe nice catch! Fixed